### PR TITLE
Add mkrwifi1010

### PR DESCRIPTION
**Pull request by jakewins**
I haven't tested this yet, but thought I should open a PR to get a sanity check before I set fire to my arduino.. 

This is based on the guide [here](https://github.com/atsamd-rs/atsamd/wiki/Adding-a-new-board), with pin assignments from [here](https://github.com/arduino/ArduinoCore-samd/blob/master/variants/mkrwifi1010/variant.cpp) and memory offsets from [here](https://github.com/arduino/ArduinoCore-samd/blob/master/variants/mkrwifi1010/linker_scripts/gcc/flash_with_bootloader.ld) thanks to @sajattack

### CODE REVIEW

File: boards/arduino_mkrwifi1010/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Adafruit Gemma M0 Board Support Crate
+
+This crate provides a type-safe API for working with the [Adafruit Gemma M0
+board](https://www.adafruit.com/product/3501).
```
Comment (jessebraham) [inline boards/arduino_mkrwifi1010/README.md:4]: The title and product link/text need to be updated here please.

Comment (jakewins) [reply boards/arduino_mkrwifi1010/README.md]: Done!

File: boards/arduino_mkrwifi1010/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Arduino MKR WiFi 1010 Board Support Crate
+
+This crate provides a type-safe API for working with the [Arduino MKR WiFi 1010 board](https://store.arduino.cc/usa/mkr-wifi-1010).
+
+## Examples
```
Comment (jakewins) [inline boards/arduino_mkrwifi1010/README.md:5]: This is straight from the Arduino Nano 33 IoT crate, I don't know if this works yet. Will update this once I've actually tested it on the real board.

**Comment by jakewins** [comment]
@jessebraham I've verified the blink example works as per the instructions in the README now; let me know if there's anything else needed on this.

**Comment by jessebraham** [comment]
Thanks, this is looking pretty good! If you're able to rebase on master and convert the PR from a draft, assuming the Travis build succeeds I'm happy to merge this.

**Comment by sajattack** [comment]
@jakewins Can you finish this up?

**Comment by sajattack** [comment]
Closes #169

**Pull request closed without merging**